app/assets/stylesheets/components/_autocomplete.scss

```diff
@@ -0,0 +1,7 @@
+@import url(asset-path("accessible-autocomplete/dist/accessible-autocomplete.min.css")); // stylelint-disable function-url-quotes
```

Comment:
If i understand right this is creating a css import? Whereas we just want the CSS file imported and save a request as per: https://github.com/alphagov/content-publisher/blob/8240b79bbfbab91e24530b7f9f178f0ef541cc71/app/assets/stylesheets/components/_autocomplete.scss#L2

We made the change in Content Publisher in: https://github.com/alphagov/content-publisher/commit/40bc73a1540f51d59fe688de7be3bbf8befe7d60

Reply:
Ah that's much cleaner, thanks, have updated.